files/build_templates/buffers_config.j2

```diff
@@ -107,7 +107,7 @@ def
 {
     "CABLE_LENGTH": {
         "AZURE": {
-    {% for port in PORT %}
+    {% for port in PORT_ALL %}
```

Comment:
You are reversing Pavel's change to only configure ports with neighbor defined. This change is not desirable because fast reboot delay. Are you able to address the issue you hit with defining neighbor links?

Reply:
check line 84, if PORT is not defined, then PORT_ALL defines all the ports. if PORT is defined, PORT_ALL = PORT.

Reply:
i do not think PORT ever means the configure ports with neighbor defined.

Reply:
I see.